app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,38 @@
  *= require font-awesome
  */
 
+#map-container { 
+  border-color: #fff;
+  border-radius: 16px 16px;
+  border-style: solid;
+  border-width: 7px;
+  box-shadow: 2px 2px 10px #B1B1B1;
```

Comment:
Color literals like `#B1B1B1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#B1B1B1` should be written as `#b1b1b1`